src/InfluxDB.Collector/Pipeline/Aggregate/AggregatePointEmitter.cs

```diff
@@ -0,0 +1,103 @@
+﻿// ==========================================================================
+// AggregatePointEmitter.cs
+// Bus Portal (busliniensuche.de)
+// ==========================================================================
+// All rights reserved.
+// ========================================================================== 
```

Comment:
Contributions need to be made under the same license as the rest of the codebase

Reply:
Done